### PR TITLE
Snellius instructions updated

### PR DESCRIPTION
**Pull request by thjsal**
Snellius installation instructions (and the example job script) update based on the recommendations and tests from Satish Kamath and @Basdorsman (to improve the performance).

Changes summarized:

- Using foss instead of intel
- Using python virtual environment (if wanted) instead of conda
- A few flags changed when installing MultiNest

### CODE REVIEW

File: docs/source/HPC_systems.rst [hunk 1]
```diff
@@ -35,78 +35,36 @@ To be additionally safe, run:
...
-    cd; wget https://github.com/mpi4py/mpi4py/releases/download/3.1.5/mpi4py-3.1.5.tar.gz
-    tar zxvf mpi4py-3.1.5.tar.gz
-    cd mpi4py-3.1.5
-    python setup.py build   --mpicc=/sw/arch/RHEL8/EB_production/2022/software/impi/2021.6.0-intel-compilers-2022.1.0/mpi/2021.6.0/bin/mpicc
-    python setup.py install
+    mkdir venvs
+    python -m venv ./venvs/xpsi_py3
 
-To test on the login node:
+To access all the loaded site packages when activating the virtual environment, one needs to modify the file ``./venvs/xpsi_py3/Pyvenv.cfg`` (using e.g. ``vim`` or ``emacs`` text editor) to change "false" into "true":
```
Comment (MHoogkamer) [inline docs/source/HPC_systems.rst:57]: Change `Pyvenv.cfg` to `pyvenv.cfg`

File: docs/source/HPC_systems.rst [hunk 2]
```diff
@@ -173,30 +125,17 @@ only need:
 .. code-block:: bash
```
Comment (MHoogkamer) [inline docs/source/HPC_systems.rst:126]: Add `git clone https://github.com/xpsi-group/xpsi.git`

File: docs/source/HPC_systems.rst (hunk 1 above)
Comment (thjsal) [reply docs/source/HPC_systems.rst]: Fixed.

File: docs/source/HPC_systems.rst (hunk 2 above)
Comment (thjsal) [reply docs/source/HPC_systems.rst:126]: Fixed.

**Review by MHoogkamer — APPROVED** [review]

**Pull request merged**